app/components/UI/Ramp/Deposit/Views/BasicInfo/BasicInfo.tsx

```diff
@@ -68,12 +72,17 @@ const BasicInfo = (): JSX.Element => {
   const dateInputRef = useRef<TextInput>(null);
   const ssnInputRef = useRef<TextInput>(null);
 
+  const utcDateToPrefill = new Date(previousFormData?.dob || '');
+  const localTimestampToUseInternally =
+    utcDateToPrefill.getTime() +
+    utcDateToPrefill.getTimezoneOffset() * 60 * 1000;
```

Comment:
Protect against NaN dates

```suggestion
  const utcDateToPrefill = new Date(previousFormData?.dob || '');
  const localTimestampToUseInternally = isNaN(utcDateToPrefill) ? '' :
    (utcDateToPrefill.getTime() +
    utcDateToPrefill.getTimezoneOffset() * 60 * 1000).toString();
```